pkg/ui/styl/shame.styl

```diff
@@ -111,7 +111,7 @@
 
   .nv-axislabel
     font-family Lato-Regular
-    fill $button-border-color
+    fill tooltip-color
```

Comment:
is this actually resolving as a variable? variables have to start w/ dollar signs AFAIK

Reply:
fixed!